app/assets/javascripts/Components/Result/url_viewer.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import "../../../stylesheets/common/_url_viewer.scss";
+
+export class URLViewer extends React.Component {
+  state = {
```

Comment:
State should be set in the `constructor` not as a static attribute on the class.